pkgs/by-name/qr/qrtool/package.nix

```diff
@@ -6,18 +6,18 @@
 , installShellFiles
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "qrtool";
-  version = "0.8.5";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
     owner = "sorairolake";
     repo = "qrtool";
-    rev = "v${version}";
-    sha256 = "sha256-jrvNZGO1VIDo6Mz3NKda1C7qZUtF9T00CAFK8yoGWjc=";
+    rev = "v0.9.0";
+    hash = "sha256-LDVYSVI7hYR9eVpq5xb1rUnT9DG4b97qHl3oy+hZu3Q=";
   };
 
-  cargoSha256 = "sha256-JOnvlabCr3fZsIIRc2qTjf50Ga83zL8Aoo2sqzMBs7g=";
+  cargoSha256 = "sha256-bB07rZ4QZ8ALIQudbJAqudxUFe68LSX6AKxNFy5xg+8=";
```

Comment:
```suggestion
  cargoHash = "sha256-bB07rZ4QZ8ALIQudbJAqudxUFe68LSX6AKxNFy5xg+8=";
```

Reply:
https://ryantm.github.io/nixpkgs/languages-frameworks/rust/ doesn't recommend one over the other.